test/beamchmark/formatters/html_test.exs

```diff
@@ -83,5 +85,29 @@ defmodule Beamchmark.Formatters.HTMLTest do
       assert File.exists?(options[:output_path])
       assert File.read!(options[:output_path]) == mock_html
     end
+
+    test "formatted_average_cpu_usage" do
```

Comment:
We try to name tests in a way that allows for reading them as `test if x does something properly`

Please, check other places too